extensions/resteasy-reactive/quarkus-resteasy-reactive-jackson/deployment/src/test/java/io/quarkus/resteasy/reactive/jackson/deployment/test/SimpleJsonResource.java

```diff
@@ -81,4 +83,30 @@ public void run() {
         }).start();
     }
 
+    @GET
+    @Path("/user-without-view")
+    public User userWithoutView() {
+        return testUser();
+    }
+
+    @JsonView(Views.Public.class)
```

Comment:
Should the view class be added for reflection?

I don't see us doing it in the `quarkus-jackson` extension and it looks like reflection could be used by Jackson, right?

Reply:
I'll check and see if it's needed

Reply:
I just tried it in native and it works without registering anything from reflection

Reply:
Hmmm. How does it work? It's not using reflection?

Reply:
Let me try something. I might have messed up my test

Reply:
Indeed it fails if I actually perform a proper test. I'll fix it.

Reply:
Properly fixed